tests/percy/index.js

```diff
@@ -17,11 +19,22 @@ PercyScript.run(async (page, percySnapshot) => {
 
   const standardPageNavigator = new StandardPageNavigator(page, `http://localhost:${PORT}`);
   const iframePageNavigator = new IframePageNavigator(page, `http://localhost:${PORT}`, 'iframe_test');
+
   const standardCamera = new Camera(percySnapshot);
   const iframeCamera = new Camera(percySnapshot, true);
 
   await (new Photographer(standardPageNavigator, standardCamera).captureSnapshots());
   await (new Photographer(iframePageNavigator, iframeCamera).captureSnapshots());
 
+  standardPageNavigator.setCurrentLocale('es');
+  iframePageNavigator.setCurrentLocale('es');
+
+  standardCamera.setLocale('es');
+  iframeCamera.setLocale('es');
+  
+  await (new MultilangPhotographer(standardPageNavigator, standardCamera, queryConfig.es).captureSnapshots());
+  await (new MultilangPhotographer(iframePageNavigator, iframeCamera, queryConfig.es).captureSnapshots());
```

Comment:
do people think it would be possible to not include the "multilang + iframe" combo? I'm imagining nearly all bugs in the iframe experience would be caught by the "en + iframe" snapshots, outside of maybe the multilang variant having longer text that causes weird wrapping/overflow bugs?

Reply:
since iframe experience utilize the same pages - no changes to translation logic and multilang template cards, it does seems redundant to repeat all multilang variant tests for iframe. Yeah, I can remove this test combo